css/properties/-webkit-text-decorations-in-effect.json

```
@@ -20,7 +20,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "≤13.1"
+              "version_added": "≤5.1"
```

Already present in Safari 4.0 (https://github.com/caugner/css-properties/commit/33ce1ea279967afb2272f8b51aa53a147afcb375), but appears to have been removed in Safari 13.1 (https://github.com/caugner/css-properties/commit/7be3974224718f9215519f2a10916d32ed306da9#diff-b63d72471087ee495d4f5c7e94b58bef200825135bdee1bdcddbfbd594b0c328L72).

```suggestion
              "version_added": "≤4",
              "version_removed": "13.1",
```